# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

AtlasSimInterfaceActionGoal action_goal
AtlasSimInterfaceActionResult action_result
AtlasSimInterfaceActionFeedback action_feedback
